test/Microsoft.ML.Tests/Scenarios/Api/Estimators/Wrappers.cs

```diff
@@ -189,10 +184,14 @@ public ScorerWrapper(IHostEnvironment env, IDataView scorer, TModel trainedModel
             : base(env, scorer)
         {
             _featureColumn = featureColumn;
-            InnerModel = trainedModel;
+            Model = trainedModel;
         }
 
-        public TModel InnerModel { get; }
+        public TModel Model { get; }
+
+        public string FeatureColumn => _featureColumn;
+
+        public ColumnType FeatureColumnType => throw new NotImplementedException();
```

Comment:
>throw new NotImplementedException(); [](start = 47, length = 36)

is this intended?

Reply:
That's ok since this wrapper is temporary.

---
In reply to: [212683771](https://github.com/dotnet/machinelearning/pull/716#discussion_r212683771) [](ancestors = 212683771)